### ISSUE TITLE
Invoices originating from an Order are not retrievable

### ISSUE THREAD

**Issue by j-mastr** [j-mastr/sevdesk-api#55]
As reported by @Cruiser13 in #54, getting an invoice `$invoiceId` that was generated out of an Order:

```
$apiInstance = new Itsmind\Sevdesk\Api\InvoiceApi(/* omitted */);

try {
    $result = $apiInstance->getInvoiceById($invoiceId);
} catch (Exception $e) {
    echo 'Exception when calling InvoiceApi->getInvoiceById: ', $e->getMessage(), PHP_EOL;
}
```

throws `Exception when calling InvoiceApi->getInvoiceById: Invalid value 'Order' for 'object_name', must be one of 'Invoice'`.

The issue is related to the `Model_InvoiceResponse/properties/origin/objectName` field.

**Comment by Cruiser13** [comment]
@j-mastr yes this is the issue we are facing at the moment.

**Comment by j-mastr** [comment]
Alright, should be fixed in 1.4.0 now 👍
